src/gui/sharelinkwidget.cpp

```diff
@@ -284,7 +289,10 @@ void ShareLinkWidget::slotCopyLinkShare(bool clicked)
 {
     Q_UNUSED(clicked);
 
-    QApplication::clipboard()->setText(_linkShare->getLink().toString());
+    QUrl link = _linkShare->getLink();
+    if (_useDirectDownload) link = _linkShare->getDirectDownloadLink();
```

Comment:
Please use curly braces on this if.

Reply:
OK, will take care of the braces.

But how do you squash commits on Github? I don't see an option for that (like on Gitlab)...

Reply:
Well you'd do it using git locally and then force push on the branch.

Reply:
git rebase -i HEAD~2